lib/diagnostic.js

```diff
@@ -5,30 +5,60 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
-
+    let result = arguments[0];
+    for (let i = 0; i < arguments.length; i++)
+    {
+        if (result > arguments[i])
+        {
+          result = arguments[i];
+        }
+    }
+    return result;
 };
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
 const sum = function sum() {
-
+  let result = 0;
+  for (let i = 0; i < arguments.length; i++)
+  {
+      result = result + arguments[i];
+  }
+  return result;
```

Comment:
Passes 2/3 tests. be sure to run `grunt test` to see what is not passing and make changes accordingly.